util/ziputil/zip_writer.go

```diff
@@ -7,12 +7,41 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"time"
 )
 
+type Options func(zw *zip.Writer) error
+
+// WithREADME returns an option to write the README file.
+func WithREADME() Options {
+	return func(zw *zip.Writer) error {
+		const downloadREADME = `
+To review the CPU profiling or heap profiling result interactively:
+
+$ go tool pprof --http=0.0.0.0:1234 cpu_xxx.proto
+`
+
+		zipFile, err := zw.CreateHeader(&zip.FileHeader{
+			Name:     "README.md",
+			Method:   zip.Deflate,
+			Modified: time.Now(),
+		})
+		if err != nil {
+			return err
+		}
+
+		_, err = zipFile.Write([]byte(downloadREADME))
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+}
```

Comment:
This is not utility at all. For example, you will never find a "zip" tool in your PC that contains a settings like [x] Add README..